terraform/modules/jenkins/ec2.tf

```diff
@@ -0,0 +1,41 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+resource "aws_instance" "jenkins2" {
+  ami                    = "ami-785db401"
```

Comment:
Is this the updated ami? I can't remember...

Reply:
This one claims to be "Canonical, Ubuntu, 16.04 LTS, amd64 xenial image build on 2017-07-21".

The old one is some custom thing from 2015. So yeah I suppose it's more up-to-date. I dunno if that's good or not really in terms of reproducibility, but so far I doubt any of the problems I've seen have been caused by OS-level stuff, so it's probably overall good.

Reply:
Does this image mitigate spectre and meltdown? I feel like they were newer than 2017?

Reply:
Have updated the box to the latest equivalent AMI, we'll see what gives...